adapters/Maio/MaioAdapter/RTBAdapter/GADRTBAdapterMaioEntryPoint.h

```diff
@@ -0,0 +1,26 @@
+//
+//  GADRTBAdapterMaio.h
+//  Adapter
+//
+//  Created by i-mobile on 2020/11/18.
+//  Copyright © 2020 Google. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <GoogleMobileAds/GoogleMobileAds.h>
+
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
remove these across all files since you are already explicitly indicating nullability.

Reply:
Certainly.